apps/monitoring/ptl-intsvc/00/kustomization.yaml

```diff
@@ -6,3 +6,4 @@ resources:
 patches:
 - path: ../../kube-prometheus-stack/ptl-intsvc/00.yaml
 - path: ../../opentelemetry-collector/ptl-intsvc/00.yaml
+- path: ../../version-reporter/helmcharts/environment-patches/sbox-intsvc.yaml
```

Comment:
sbox-intsvc on ptl?

Reply:
this is ptl-intsvc so ptl sbox

Reply:
They arent the same, there is ptl-intsvc and sbox-intsvc (which is ptlsbox)

Reply:
link to this https://github.com/hmcts/cnp-flux-config/blob/master/apps/monitoring/version-reporter/helmcharts/environment-patches/sbox-intsvc.yaml